### ISSUE TITLE
Clarify CP/CPS disclosure requirements for cross-certificates

### ISSUE THREAD

**Issue by wthayer** [mozilla/www.ccadb.org#37]
As [discussed on m.d.s.p.](https://groups.google.com/d/msg/mozilla.dev.security.policy/89iF_4Ovpwg/eGd7ukOOCwAJ), when a CA certificate appears on the audit statement of both the issuer and subject and those are two different organizations, expectations for CP/CPS disclosure are unclear. We may want to require information for both the cert and the Subject + SPKI, or decide which one is appropriate. I think the CP/CPS of the CA in possession of the private key (i.e. the Subject of the cert) is the appropriate one to disclose for these certificates. However, the responsibility of disclosing falls on the Issuer.

**Comment by ryancdickson** [comment]
I think this is overcome by events?

The CCADB Policy specifies requirements for:
-  [CA certificate disclosures](https://www.ccadb.org/policy#3-certificate-disclosures)
- [Policy disclosures](https://www.ccadb.org/policy#4-policy-disclosures)
- [Audit disclosures](https://www.ccadb.org/policy#5-audit-disclosures)

Several reports now exist on the CA Owner's CCADB "CA Task List" that also highlight missing information and issues.

If anyone feels differently, please reopen.

**Comment by AGWA** [comment]
@ryancdickson I don't believe this has been fixed, as I don't see any clarification about which CP/CPS to disclose when the Issuer and Subject of a CA certificate are different organizations.

For context, this issue was sparked by [this incident report](https://bugzilla.mozilla.org/show_bug.cgi?id=1567061) in which I observed that GoDaddy had disclosed that two certificates (records 001o000000ntX22AAE and 001o000000ervM0AAI) operate under a GoDaddy CP/CPS even though the private key is held by Amazon Trust Services. The records still say this. This is rather unhelpful, and for the last 6 years I've had to special case these two records (and only these two records, since no other CA has done this) when processing CCADB reports.

I believe this should be reopened until the CCADB policy is updated to say that the CP/CPS of the CA in possession of the private key (i.e. the Subject of the cert) is the appropriate one to disclose.
